README.md

```diff
@@ -63,11 +65,20 @@ valence.
 
 For a detailed introduction, please check out the [notebook](./notebook.ipynb).
 
-```
+```bash
 $ pip install -r requirements.txt
 $ jupyter notebook notebook.ipynb 
 ```
 
 ## Citation
 
-TBA
+If you use our model in your own work, please cite the following paper:
+
+```bibtex
+@article{wagner2022dawn,
+  title={Dawn of the transformer era in speech emotion recognition: closing the valence gap},
+  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Eyben, Florian and Schuller, Bj{\"o}rn},
```

Comment:
```suggestion
  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Eyben, Florian and Schuller, Bj{\"o}rn W., and Burkhardt, Felix},
```

Reply:
Besides adding Felix, I also added "W." to Björns name as he used it in the paper.